app/models/likes.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :likes_counter
```

Comment:
- [ ] Please add a method here that updates the number of likes as mentioned in the requirements. It can look something like this.
```
def update_likes_counter
  post.increment!(:likes_counter)
end
```

Reply:
Done! 😄

Reply:
- Well resolved 👍🏽